pisa/stages/flux/mceq_barr.py

```diff
@@ -94,6 +95,7 @@ def __init__(
         table_file,
         include_nutau_flux=False,
         use_honda_nominal_flux=True,
+        use_relative_gradients=False,
```

Comment:
Documentation of this parameter is missing.

Reply:
Added now